src/sentry/static/sentry/app/views/organizationEventsV2/eventDetails.jsx

```diff
@@ -67,13 +69,18 @@ const ModalContainer = styled('div')`
   z-index: ${p => p.theme.zIndex.modal};
 `;
 
-const CloseButton = styled(Button)`
-  position: absolute;
-  top: -10px;
-  right: -10px;
-  padding: 10px;
+const CloseButton = styled('button')`
+  background: #fff;
   border-radius: 20px;
+  line-height: 12px;
+  height: 30px;
   box-shadow: ${p => p.theme.dropShadowLight};
+  border: 1px solid ${p => p.theme.borderDark};
+
+  position: absolute;
+  top: -15px;
+  right: -15px;
```

Comment:
Is it possible to use percentages here (and maybe for border-radius as well) so if we change height things would scale accordingly?

Reply:
I've made the close button a function component that allows all the sizing/positioning to be driven by props.

Reply:
Nice